ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_linalg.py

```diff
@@ -217,3 +217,42 @@ def test_kron(
         a=x[0],
         b=x[1],
     )
+
+
+# matrix_exp
+@handle_test(
+    fn_tree="functional.experimental.matrix_exp",
+    dtype_x=helpers.dtype_and_values(
```

Comment:
E   RuntimeError: linalg.matrix_exp: The input tensor must have at least 2 dimensions.

Reply:
in addition: E   tensorflow.python.framework.errors_impl.InvalidArgumentError: Value for attr 'T' of uint8 is not in the list of allowed values: bfloat16, half, float, double, int8, int16, int32, int64

Reply:
also, I suspect you uses tensorflow's function in your numpy backend version of the function, which is not correct.